data/abilities.js

```diff
@@ -728,10 +725,7 @@ let BattleAbilities = {
 		onUpdate: function (pokemon) {
 			if (['mimikyu', 'mimikyutotem'].includes(pokemon.template.speciesid) && this.effectData.busted) {
 				let template = this.getTemplate(pokemon.template.speciesid === 'mimikyutotem' ? 'Mimikyu-Busted-Totem' : 'Mimikyu-Busted');
-				pokemon.formeChange(template);
-				pokemon.baseTemplate = template;
-				pokemon.details = template.species + (pokemon.level === 100 ? '' : ', L' + pokemon.level) + (pokemon.gender === '' ? '' : ', ' + pokemon.gender) + (pokemon.set.shiny ? ', shiny' : '');
-				this.add('detailschange', pokemon, pokemon.details);
+				pokemon.formeChange(template, this.effect, true);
```

Comment:
(Since `formeChange` takes a templateid, you don't need to call `this.getTemplate` manually any more. Similarly for the Primals.)